src/components/ClaudyMenu.svelte

```diff
@@ -54,8 +71,32 @@
       return {
         openedAction: false,
         selectedAction: null,
+        selectedActionUrl: '',
         getIcon (iconName) {
           return require(`../assets/icons/claudyActions/${iconName}`)
+        },
+        computeUrl (action) {
+          const appsList = this.appsList
+          if (action.link.type === 'apps' && action.link.appSlug) {
+            if (!appsList) {
+              console.warn('No apps found on the Cozy')
```

Comment:
Since the action button can be disabled for this reason I would prefer to warn it here. At least until we have a link to the store to put instead. What do you think?